ansible/ansible_collections/agentic/agent_configs/plugins/modules/deepseek_tui_hook.py

```
@@ -0,0 +1,500 @@
+    had_hook_entries: bool
+
+
+def _had_hook_entries_before(data: dict[str, Any]) -> bool:
+    """Return whether the TOML data already contained a hooks.hooks list before mutation."""
+    hooks = data.get("hooks")
+    return isinstance(hooks, dict) and "hooks" in hooks
+
+
+def _cleanup_absent_noop(
+    state: str,
+    changed: bool,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)